wilderness-course-ticket-reminder plugin

neafey opened the pull request:
Plugin that reminds the player to claim a ticket after completing a lap in the Wilderness Agility Course.
Highlights the dispenser after completing a lap to help player remember to claim your ticket.
Also changes the default left-click action on the pipe to "Walk here" if no ticket has been claimed.

LlemonDuck commented:
* You should remove the unused TicketReminderConfig. You can re-add it later if you need to introduce config options.
* Objects#requireNonNull is an assertion, not a guarantee, and does not prevent NPEs. Uou're leaking these onto the client thread. Since it _can_ actually be null, you should check for such yourself.

neafey commented:
Thank you for the feedback! This is my first plugin, so I was a bit unsure about whether to remove the config if it wasn’t used. I’ve removed it now.

You’re absolutely right about Objects#requireNonNull. I’ve replaced it with a proper null check instead.

I hope this addresses everything, but I’m happy to make more changes if needed to get the PR accepted.
